java/src/main/java/ai/rapids/cudf/ColumnWriterOptions.java

```diff
@@ -33,9 +33,13 @@ public class ColumnWriterOptions {
   private boolean isNullable;
   private boolean isMap = false;
   private String columnName;
+  // only for Parquet
+  private int parquetFieldId;
```

Comment:
How does one specify they do _not_ want a field ID written for this column?  It looks like we're always going to write everything with a field ID of zero unless something is specified.  We should only specify a field ID to Parquet when it is specified as a column writer option explicitly.  I'm not sure if there's a safe sentinel value to use here, so we may need another boolean to indicate whether this field has been set and is valid to use.

Reply:
Negative field ID is valid, so should add another boolean as you mentioned.

Reply:
done.